### PR TITLE
feat(controller): support movies endpoint, DB creation and tests

### PR DESCRIPTION
**Pull request by tzaitsev**

### CODE REVIEW

File: test/plugins/features/movies/index.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const Movies = require('../../../../lib/server');
+
+describe('movies integration', () => {
+
+  describe('create', () => {
+
+    it('creates a movie', () => {
```
Comment (kylekwong) [inline test/plugins/features/movies/index.test.js]: Can you make this use async/await?

File: test/plugins/features/movies/index.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+      return Movies.inject({
+        url: '/movies',
+        method: 'POST',
+        payload
+      }).then((response) => {
+        expect(response.statusCode).to.eql(200);
+        expect(response.result.object).to.eql('movie');
+        expect(response.result.title).to.eql(payload.title);
+        expect(response.result.release_year).to.eql(null);
```
Comment (kylekwong) [inline test/plugins/features/movies/index.test.js]: nit: `to.be.null;`

**Review by kylekwong — APPROVED** [review]

File: lib/plugins/features/movies/index.js [hunk 3]
```diff
@@ -1,13 +1,19 @@
 'use strict';
 
+const MovieValidator = require('../../../validators/movie');
+const Controller = require('./controller');
```
Comment (ajpawlicki) [inline lib/plugins/features/movies/index.js]: nit: we usually align these i.e.

```node
const MovieValidator = require('../../../validators/movie');
const Controller     = require('./controller');
```

**Review by ajpawlicki — APPROVED** [review]

File: lib/plugins/features/movies/index.js (hunk 3 above)
Comment (tzaitsev) [reply lib/plugins/features/movies/index.js]: Done.

**Pull request merged**